README.md

```diff
@@ -273,6 +273,11 @@ To clean them up automatically, you may:
   enable --now pikaur-cache.timer`.
 
 
+##### How to restore original PKGBUILD after editing?
+
+`rm -r ~/.local/share/pikaur/aur_repos/${PACKAGE_NAME}`
```

Comment:
thanks!

mb also mention non-destructive way as alternative?
```
cd ~/.local/share/pikaur/aur_repos/${PACKAGE_NAME}
git diff
git checkout -- '*'
```

Reply:
actually i've just realized what `rm ...` is not a way to go at all, because in that case you'll also remove `last_installed.txt` which contains last build hash and needed for asking to review package changes

you can learn more about that in the readme: https://github.com/actionless/pikaur#directories

Reply:
Ah, that's true. Is the `git` method you mentioned still okay then, or would that also mess with `last_installed.txt`?

Reply:
the git method is "official" one :) it's the way how it mentioned under the hood of the program.